### PR TITLE
Add initial polyfill draft

### PR DESCRIPTION
**Pull request by legendecas**

### CODE REVIEW

File: spec.emu [hunk 1]
```diff
@@ -81,7 +81,7 @@ contributors: Chengzhong Wu, Hemanth HM, Ecma International
         <ins class="block">
           <emu-clause id="sec-error.prototype.cause">
             <h1>Error.prototype.cause</h1>
-            <p>The initial value of `Error.prototype.cause` is the empty String.</p>
+            <p>The initial value of `Error.prototype.cause` is *undefined*.</p>
```
Comment (hemanth) [inline spec.emu]: In #6 we decided it to be an empty string right?

Comment (legendecas) [reply spec.emu]: Yeah, but when I'm writing the polyfill and re-thinking about the default value about the following points: 

1. value of `error.cause` can be an arbitrary javascript value and most likely an Error instance.
2. how to identify no-cause specified.

For point 1, I believe the original thought of the default value to be an empty string is to align with the `Error.prototype.message`. However, `Error.prototype.message` can only be strings as the step 19.5.1.1.3.a cast an op ToString to the `message`. This is different from `Error.prototype.cause`.

For point 2, I'm considering we can no longer assume the value of `error.cause` to be existing for not [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) (undefined, null, empty string, etc.) for point 1.

So `undefined` may be more suitable as the default value for `Error.prototype.cause` here.

Comment (ljharb) [reply spec.emu]: However, it's likely to be stringified into the error toString - and a string of `undefined` showing up would be very bad.

Comment (legendecas) [reply spec.emu]: Fair point. Will do a revert.

**Pull request merged**